openreview/venue_request/process/deployProcess.py

```diff
@@ -404,3 +404,59 @@ def process(client, note, invitation):
     for reply in replies:
         reply.readers = readers
         client.post_note(reply)
+
+    client.post_invitation(openreview.Invitation(
+        id=SUPPORT_GROUP + '/-/Request' + str(forum.number) + '/Recruitment_Status',
+        super=SUPPORT_GROUP + '/-/Recruitment_Status',
+        reply={
+            'forum': forum.id,
+            'referent': forum.id,
```

Comment:
referent shouldn't be here. The referent value is used when we want to revise the request form note. In this case, we want to make replies.

Reply:
Done.